app/assets/javascripts/ideas.js

```diff
@@ -0,0 +1,20 @@
+$( document ).ready(function() {
+  $('tbody tr .idea-destroy').on('click', function() {
+    $idea = $(this).parent();
+    ideaId =  $idea.data("id");
```

Comment:
Mixed double and single quotes.<br>'ideaId' is not defined.<br>'$idea' is not defined.